### PR TITLE
Adding /map/svg - close #43

### LINKED ISSUE #43: [Feature] Create a static image for the needs of volunteers
We would like to post on a regular basis (probably weekly) a static image on our social networks about where we need volunteers depending on our database. For example:

https://twitter.com/en1ereligne/status/1243157326447742976

Having a generator of such image in the back-office would help us a lot.

### PR DESCRIPTION
**Pull request by Lucas-C**
Preview (with randomly colored departments):

enpremiereligne-map-random

Remain to do before merging:
- [x] add some tests
- [ ] gérer les départements d'outre-mer et leurs codes ZIP
```
971 | Guadeloupe
972 | Martinique
973 | Guyane
974 | La Réunion
976 | Mayotte
```

Could I have some feedback on the approach taken please ?
Does this look right to you ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .php_cs.cache
 docker-compose.override.yaml
 
+# IntelliJ project files:
```
Comment (tgalopin) [inline .gitignore:4]: You should not put these files in projects gitignore but in a global gitignore on your system.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

File: src/Controller/MapController.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+     */
+    public function svg(StatisticsAggregator $statsAgg, Request $request)
+    {
+        $threshold = $request->query->getInt('threshold', self::DEPARTMENT_COLORING_THRESHOLD);
+
+        $dynamicStyle = '';
+        $fillColor = self::DEPARTMENT_FILL_COLOR;
+        $ownerByDepartment = $statsAgg->countOwnersByDepartment();
+        // Test code to get half of the departments randomly colored on the map:
+        /*$ownerByDepartment = [];
```
Comment (tgalopin) [inline src/Controller/MapController.php:40]: Can you remove the commented code?

File: .gitignore (hunk 1 above)
Comment (Lucas-C) [reply .gitignore:4]: Ok !

File: src/Controller/MapController.php (hunk 2 above)
Comment (Lucas-C) [reply src/Controller/MapController.php:40]: Done

**Comment by Lucas-C** [comment]
I think this can be merged, unless you want France overseas departments included (I'm having difficulties finding a SVG map of them free of rights)

By the way, do not hesitate to submit more feature requests so that I can help ! ;)

**Comment by tgalopin** [comment]
Thank you @Lucas-C !

**Pull request merged**